src/packages/__VUE/checkbox/index.taro.vue

```diff
@@ -40,6 +41,7 @@ export default create({
   },
   emits: ['change', 'update:modelValue'],
   setup(props, { emit, slots }) {
+    const disabled = useFormDisabled(props.disabled);
```

Comment:
将 `props.disabled` 直接作为函数的参数传递，会导致其丢失响应性，参考 [vue/props](https://cn.vuejs.org/api/composition-api-setup.html#accessing-props)
```suggestion
    const disabled = useFormDisabled(toRef(props, 'disabled'));
```
可以增加 toRef 处理，并设置 useFormDisabled 的参数类型为 `Ref<boolean>`